src/connector.rs

```diff
@@ -45,12 +48,23 @@ where
     })
 }
 
+/// not implemented for MacOS
```

Comment:
Would it be possible to explain why this is so, and link back to #43? That way future users can see why they're getting panics and what they'd need to do to fix.